deploy/mpi-operator.yaml

```diff
@@ -71,7 +71,7 @@ rules:
   resources:
   - pods
   verbs:
-  - get
+  - get,list,watch
```

Comment:
Could you split these into multiple lines like others?

Reply:
No problem.